This file makes repeated reference to E* constants, and therefore must
include errno.h.  It's possible that errno.h is provided by one of the
other headers, but it's best practice to include it locally regardless.

Kerberos RT #3266

--- krb5-1.4.2.orig/src/kdc/kerberos_v4.c
+++ krb5-1.4.2/src/kdc/kerberos_v4.c
@@ -141,6 +141,7 @@
  */
 #include "com_err.h"
 #include "extern.h"		/* to pick up master_princ */
+#include <errno.h>
 
 static krb5_data *response;
 
